src/utilities.ts

```diff
@@ -51,9 +45,13 @@ export function convertTimeStampToTime(elapsedTime: number) {
     return time;
 }
 
-export function formatDuration(time: Time) {
-    const hours = `${time.hours === "00" ? "" : time.hours[0] === "0" ? time.hours[1] + " h" : time.hours + " h"}`;
-    const minutes = `${time.minutes === "00" ? "0 min" : time.minutes[0] === "0" ? time.minutes[1] + " min" : time.minutes + " min"}`;
+export function formatDuration(time: number) {
+    const sec = time / 1000;
+
+    const hours = Math.floor(sec / (60 * 60));
+    const h = hours > 0 ? `${hours} h` : "";
+    const minutes = Math.floor((sec / 60) % 60);
+    const min = minutes > 0 ? `${minutes} min` : "";
```

Comment:
Calculations logic is duplicated between these two functions.  You should avoid having duplicated code, re-write these functions in a way that you only calculate hours and minutes once.